plugins/registry.js

```diff
@@ -47,6 +53,12 @@ export default class PluginRegistry {
         this.id = id;
     }
 
+    openInteractiveDialog(dialog) {
```

Comment:
I actually don't think this makes sense as part of the registry since we're not actually registering anything here. What about instead exporting it as a function from `plugins/export.js`? @marianunez recently did something similar to expose a function to plugins for opening the RHS

Reply:
In [this implementation](https://github.com/mattermost/mattermost-webapp/pull/2827) (still in progress), we are going with the approach of returning the action that triggers opening the RHS to the client plugin when they call the regiterRHS component rather than exposing it at the export.js. Then the client plugin will dispatch this action when they need to.  

You can also see and example on the [demo plugin](https://github.com/mattermost/mattermost-plugin-demo/pull/31)

Reply:
@jwilander I changed to exporting the function from plugins/export.js.  Is this an ok implementation?

here is an example of using the function
https://github.com/scottleedavis/mattermost-plugin-remind/blob/issue_73_eval/webapp/src/actions.js#L85

very inspiring @marianunez, I learned somethings!  In this case since the InteractiveDialog is less involved, I went with export.js, so webapp plugins can call it where ever needed.